example/mini_core_hello_world.rs

```diff
@@ -168,6 +168,11 @@ fn main() {
         world as Box<dyn SomeTrait>;
 
         assert_eq!(intrinsics::bitreverse(0b10101000u8), 0b00010101u8);
+        assert_eq!(intrinsics::bitreverse(0xddccu16), 0x33bbu16);
+        assert_eq!(intrinsics::bitreverse(0xffee_ddccu32), 0x33bb77ffu32);
+        assert_eq!(intrinsics::bitreverse(0x1234_5678_ffee_ddccu64), 0x33bb77ff1e6a2c48u64);
+        // != cannot be applied to type u128?
```

Comment:
You probably need to add an `impl PartialEq for u128` in `example/mini_core.rs`.

Reply:
That fixes it, thanks.